.github/actions/run_opencl_cts/action.yml

```diff
@@ -0,0 +1,45 @@
+name: run opencl cts
+description: run opencl cts
+
+# This action is not standalone and assumes it has been run after the build_opencl_cts action.
+
+inputs:
+  target:
+    description: 'target architecture'
+  test_type:
+    description: 'quick | full'
+    default: "quick"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Download ock artefact
+      uses: actions/download-artifact@v4
+      with:
+        name: ock_${{inputs.target}}
+        path: install_ock
+
+    - name: Run opencl cts
+      shell: bash
+      env:
+        CTS_CSV_FILE: opencl_conformance_tests_${{inputs.test_type}}.csv
+        # TODO: host-x86_64-linux filter - expand for other targets
+        CTS_FILTER: cts-3.0-online-ignore-linux-host.csv
+        CTS_TIMEOUT: 18:00:00 # OK for github?
+      run: |
+        cd "$GITHUB_WORKSPACE/OpenCL-CTS"
+        echo "Running OpenCL CTS tests with CTS file $CTS_CSV_FILE with filter $CTS_FILTER"
+        # Note: the following inputs (offered on Gitlab, used by run_cities) are not provided
+        # for current targets:
+        #   inputs.cts_profile
+        #   inputs.cities_args
+        set -x
+        python -u "$GITHUB_WORKSPACE/scripts/testing/run_cities.py" -v \
```

Comment:
could we pass a disabled filter (using -d) - this could be an input so we can pass different ones in - by default pass in a default file here with test_api in it and a comment linking to a github issue.